core/src/test/java/org/apache/iceberg/encryption/envelope/KeyStoreKmsClient.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.iceberg.encryption.kms;
+package org.apache.iceberg.encryption.envelope;
```

Comment:
Can you move this into `encryption` instead? I think we want to keep most of it in the same package.